releases/release-1.22/README.md

```diff
@@ -62,7 +62,7 @@ The 1.22 release cycle is proposed as follows:
 | **Call for [Exceptions][Exception]** | Lead | Mon June 28 | week 10 | |
 | Brace Yourself, Code Freeze is Coming | Comms / Bug Triage | Mon June 28 | week 10 | |
 | **Begin Feature blog freeze** (23:59 PDT) | Comms Lead | Mon June 28 | week 10 | |
-| 1.22.0-beta.1 released | Branch Manager (TBD) | Thu June 8 | week 11 | |
+| 1.22.0-beta.1 released | Branch Manager (TBD) | Thur July 8 | week 11 | |
```

Comment:
```suggestion
| 1.22.0-beta.1 released | Branch Manager (TBD) | Thu July 8 | week 11 | |
```

Reply:
on it

Reply:
@saschagrunert is it intentional that `beta.2` comes just 5 days later?

Reply:
Yes we stuck that one in there intentionally to address a request from the community to have another beta after code freeze. But dates were tight and some were already fixed.